test/crowdsale/ZxcCrowdsale.test.js

```diff
@@ -733,6 +739,52 @@ contract('crowdsale/ZxcCrowdsale', (accounts) => {
       assert.strictEqual(actualTokens.toString(), expectedTokens.add(expectedBonus).toString());
     });
 
+    it('buyTokens should fail purchasing tokens if buyers kyc token is level 1 in presale', async () => {
```

Comment:
you need to update test descriptions for other presale tests - to include KYC level information

Reply:
https://github.com/0xcert/ethereum-crowdsale/pull/7/commits/090f7533070cb6977cd8918bd05b17c54b104447